### PR TITLE
network: downgrade log level and fix typo

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: src/network/networkd-ndisc.c [hunk 1]
```diff
@@ -558,7 +558,7 @@ static int ndisc_router_process_default(Link *link, sd_ndisc_router *rt) {
                 if (!route_gw->protocol_set)
                         route_gw->protocol = RTPROT_RA;
                 if (!route_gw->pref_set)
-                        route->pref = preference;
+                        route_gw->pref = preference;
```
Comment (keszybz) [inline src/network/networkd-ndisc.c:561]: Ouch.

Comment (keszybz) [reply src/network/networkd-ndisc.c:561]: I think it'd be better to not use titles like "fix typo" for logic errors. Strictly speaking many nasty bugs are "typos" in some sense… Anyone looking just at the log will likely skip this commit as "not interesting".

**Comment by keszybz** [comment]
LGTM too.

File: src/network/networkd-ndisc.c (hunk 1 above)
Comment (yuwata) [reply src/network/networkd-ndisc.c:561]: Thanks. Commit message is updated.

**Comment by yuwata** [comment]
Only commit message of the last commit is updated. Already CIs pass previously. Merging.

**Pull request merged**